setup.py

```diff
@@ -20,10 +23,24 @@ def get_long_desc():
     desc += open('CHANGES').read()
     return desc
 
+# foolproof way of single-sourcing version as per
+# http://stackoverflow.com/a/24517154/112943
```

Comment:
This comment is now out of date. The method at the link uses `exec` but this code no longer does.